internal/cmd/gocdk/apply.go

```diff
@@ -90,8 +93,10 @@ func ensureTerraformInit(ctx context.Context, pctx *processContext, moduleRoot,
 	}
 
 	// Biome exists but not initialized. Need to run terraform init.
-	c := exec.CommandContext(ctx, "terraform", "init")
+	c := exec.CommandContext(ctx, "terraform", "init", "-input="+strconv.FormatBool(input))
```

Comment:
Curious: what might `terraform init` prompt for? I've never seen it ask for anything.

Reply:
It can happen in more complex Terraform configurations where the remote state backend configuration is only partially configured: https://www.terraform.io/docs/backends/config.html#partial-configuration